src/components/OfflineWithFeedback.js

```diff
@@ -74,10 +74,7 @@ function applyStrikeThrough(children) {
 
 const OfflineWithFeedback = (props) => {
     const hasErrors = !_.isEmpty(props.errors);
-    const isOfflinePendingAction = props.network.isOffline && props.pendingAction;
-    const isUpdateOrDeleteError = hasErrors && (props.pendingAction === 'delete' || props.pendingAction === 'update');
-    const isAddError = hasErrors && props.pendingAction === 'add';
-    const needsOpacity = (isOfflinePendingAction && !isUpdateOrDeleteError) || isAddError;
+    const needsOpacity = props.network.isOffline && props.pendingAction === 'add';
```

Comment:
`needsOpacity` condition should include `hasErrors`, no?

Reply:
Also, I can't see opacity when trying to add member offline. I seems tricky to debug as browser's react debugger is not updating when I'm offline and when I turn online there is no pendingAction 😅

Reply:
> needsOpacity condition should include hasErrors, no?

I'm unsure but looking at the previous logic I would say no because there are three types or errors - add, delete & update. For add, regardless if its error or not I'm showing opacity. In the previous code,
> const needsOpacity = (isOfflinePendingAction && !isUpdateOrDeleteError) || isAddError;

needsOpacity is false for update or delete errors when offline. So I removed that since it does show opacity in that flow anyway. But either way I've asked for logic confirmation to ioni above so maybe its easier to keep that discussion in a single thread above?

Reply:
updated to revert the logic cause after discussing with ioni I realized it just works when adding anyway.